content/spin/csharp-components.md

```diff
@@ -0,0 +1,100 @@
+title = "Building Spin Components in C#"
+template = "spin_main"
+date = "2022-03-14T00:22:56Z"
+[extra]
+
+---
+- [Prerequisites](#prerequisites)
+- [C# HTTP Handler](#c-http-handler)
+
+C# is one of the most widely used programming languages. Therefore, Spin provides support for building WebAssembly (Wasm) components in C# through an experimental .NET SDK. This allows you to build web applications, send HTTP requests and read/update databases. 
+
+This guide assumes you are familiar with the C# programming language and the concept of WebAssembly. If you are interested in reading more about .NET and Wasm please also see our blog article [WebAssembly for .NET Developers: Introducing the Spin .NET SDK](https://www.fermyon.com/blog/webassembly-for-dotnet-developers-spin-sdk-intro).
+
+### Prerequisites
+
+To create a new .NET application, you’ll first need to install .NET 7 Preview 5 or above, Wizer and the Spin `http-csharp` template. Ensure Wizer is on your place or install it using `cargo install wizer --all-features`. Additionally, ensure you have the latest version of [spin installed](./install.md).
```

Comment:
This paragraph felt quite compressed.  You expand on the specific template install topic below, but the others are all packed in.  Might it work better as a checklist?